gigareview/console/tests/functional/EMReportJobCest.php

```diff
@@ -13,33 +13,46 @@ public function _before(FunctionalTester $I)
     }
 
     // tests
-    public function tryToPushManuscriptQueueJobToTable(FunctionalTester $I)
+    public function tryToPushManuscriptQueueJobToManuscriptTableAndUpdateStatusInIngestTable(FunctionalTester $I)
     {
         $I->runShellCommand("./yii_test fetch-reports/fetch", false);
+        $I->canSeeInDatabase("ingest", ["file_name"=>"Report-GIGA-em-manuscripts-latest-214-20220607004243.csv", "report_type"=>1, "fetch_status"=>3, "parse_status"=>null, "store_status"=>null, "remote_file_status"=>null]);
```

Comment:
Hi @kencho51 

don't use [magic numbers](https://en.wikipedia.org/wiki/Magic_number_(programming)#Unnamed_numerical_constants) in your test code. Instead use the constants defined in the Ingest class. It will make the code easier to read and understand and will avoid us having to change things in many place if we decide to change those numbers at some point.